### DISCUSSION TITLE
Problem with Aeotec - ZWA011-C02 (door/window sensor 7)

### DISCUSSION THREAD

**Discussion by Verbalinsurection** [zwave-js/zwave-js#5033] [Q&A] [answered]
Hi all,

Home Assistant : **2022.9.1**
Zwave to MQTT : **0.46.1**
Zwave-js : **10.0.4**
Device : **AEOTEC ZWA011-C02**

I have two ZWA011-C02. First, 2 weeks ago : no problem, all working good and I have a beautifull binary sensor reporting open or close (binary_sensor.fenetre_chambre_access_control_window_door_is_open).
The second today: associated with no problem, ping work, upload information working good and open / close information is visible on log, but no binary sensor on Hass : 
```2022-09-09 14:11:28.138 INFO ZWAVE: Node 33: value updated: 113-0-alarmType 0 => 0
2022-09-09 14:11:28.139 INFO ZWAVE: Node 33: value updated: 113-0-alarmLevel 0 => 0
2022-09-09 14:11:28.142 INFO ZWAVE: Node 33: value updated: 113-0-Access Control-Door state 23 => 22
2022-09-09 14:11:28.552 INFO ZWAVE: Node 33: value updated: 113-0-alarmType 0 => 0
2022-09-09 14:11:28.555 INFO ZWAVE: Node 33: value updated: 113-0-alarmLevel 0 => 0
2022-09-09 14:11:28.558 INFO ZWAVE: Node 33: value updated: 113-0-Access Control-Door state 22 => 23
```
On the Hass equipment I have Home Security - Motion detection, Firmware etc.
I have a sensor.fenetre_salon_access_control_door_state deactivated and if I force activation, I just have the stock value 22 or "Window/door is closed" but no binary sensor like the first one.

In the config I have all the same informations with binary sensors category [33-48-0-Door/Window] Sensor state (Door/Window)
I try 4 times to exlude / include, but nothing change. The binary sensor show up in log of zave-js : 
```2022-09-09T12:09:46.745Z CNTRLR   finished adding node 33:
                                    basic device class:    Routing Slave
                                    generic device class:  Notification Sensor
                                    specific device class: Notification Sensor
                                    supported CCs: 
                                    · Z-Wave Plus Info (0x5e)
                                    · Association (0x85)
                                    · Multi Channel Association (0x8e)
                                    · Association Group Information (0x59)
                                    · Transport Service (0x55)
                                    · Version (0x86)
                                    · Manufacturer Specific (0x72)
                                    · Device Reset Locally (0x5a)
                                    · Indicator (0x87)
                                    · Powerlevel (0x73)
                                    · Battery (0x80)
                                    · Security 2 (0x9f)
                                    · Binary Sensor (0x30)
                                    · Notification (0x71)
                                    · Configuration (0x70)
                                    · Wake Up (0x84)
                                    · Supervision (0x6c)
                                    · Firmware Update Meta Data (0x7a)
                                    controlled CCs:
```

I add the zwavejs log, zwavetomqtt log and zwave dump when I add the device and use : https://gist.github.com/Verbalinsurection/c07007f3f189bf22a1d3045e60462d6b

I have the same problem this week with a 
If anyone have any idea or if I mak an error, please feel free :)

**Comment by kpine** [comment, marked as answer]
See https://github.com/zwave-js/node-zwave-js/issues/5027

**Reply by Verbalinsurection** [reply]
Thabk you ! 
and sorry I've search with ZWA011 :(
